python/src/aac/plugins/first_party/lsp_server/providers/goto_definition_provider.py

```diff
@@ -77,9 +77,7 @@ def get_definition_location(self, name: str) -> list[Location]:
                 locations.extend(self.get_enum_value_lexeme_location(definition_to_find, name))
 
         if SymbolType.ROOT_KEY in symbol_types:
-            definition_to_find = language_context.get_root_keys_definition()
-            if definition_to_find:
-                locations.extend(self.get_root_key_definition_lexeme_location(definition_to_find, name))
+            logging.warn("Legacy Root key no longer exists and has no lexeme.")
```

Comment:
I couldn't quickly update this LSP feature, so I've captured it in a bug report here: https://github.com/DevOps-MBSE/AaC/issues/686

Reply:
Sounds good.